src/Core/Module/ModuleRepository.php

```diff
@@ -378,4 +381,24 @@ protected function enrichModuleAttributesFromHook(Module $module): ModuleInterfa
 
         return $module;
     }
+
+    /**
+     * Filter modules if the current employee has the right to see it
+     */
+    protected function filterModulesFromPermissions(ModuleCollection $modules): ModuleCollection
```

Comment:
```suggestion
    protected function filterModulesByPermissions(ModuleCollection $modules): ModuleCollection
```
Isn't this more logical name?

Reply:
Fixed.